commands/new_function.go

```diff
@@ -45,14 +45,20 @@ language or type in --list for a list of languages available.`,
 
 func runNewFunction(cmd *cobra.Command, args []string) {
 	if list == true {
+		var available_templates string
+
+		if f, err := ioutil.ReadDir(templateDirectory); err != nil {
+			available_templates = "There is no available languages"
+		} else {
+			for _, file := range f {
+				if file.IsDir() {
+					available_templates += fmt.Sprintf("- %s\n", file.Name())
```

Comment:
Build up an array in a separate function then return this - and have the formatting/printing separate.